components/Footer.tsx

```diff
@@ -0,0 +1,46 @@
+import Link from 'next/link';
+
+export default function Footer() {
+  return (
+    <footer className="footer">
+      <div className="gel-wrap">
+        <div className="footer__social">
+          <div className="footer__social__item">
+            <a
+              href="https://www.facebook.com/roseatecards"
+              target="_blank"
+              rel="noopener noreferrer"
+              className="footer__link footer__link--social"
+            >
+              Facebook
+          </a>
+          </div>
+          <div className="footer__social__item">
+            <a
+              href="https://www.instagram.com/roseatecards/"
+              target="_blank"
+              rel="noopener noreferrer"
+              className="footer__link footer__link--social"
```

Comment:
Why is the `footer__link--social` class needed when it's wrapped in `footer__social__item` already?

Reply:
Changed.